apps/system/iperf/iperf_net.c

```diff
@@ -51,6 +51,7 @@
 #include <netinet/in.h>
 #include <assert.h>
 #include <netdb.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?